pack3d/model.go

```diff
@@ -7,19 +7,28 @@ import (
 	"github.com/fogleman/fauxgl"
 )
 
-var Rotations []fauxgl.Matrix
+var AxisXRotations []fauxgl.Matrix
+var AxisYRotations []fauxgl.Matrix
+var AxisZRotations []fauxgl.Matrix
 
 /*The loop runs 24 times for all the rotation possibility*/
 func init() {
 	axisDirections := [2]int{-1, 1}
 	for i := 0; i < 4; i++ { // every axis 4 times to return to the original position
-		for _, s := range axisDirections{  // switch axis direction - or +
+		for _, s := range axisDirections { // switch axis direction - or +
 			for a := 1; a <= 3; a++ { // switch axis (3 axis)
-				up := AxisZ.Vector() // z axis
+				up := AxisZ.Vector()                                  // z axis
 				m := fauxgl.Rotate(up, float64(i)*fauxgl.Radians(90)) // Rotation matrix in z axis (4 by 4 matrix)
 				//fmt.Println(Axis(a).Vector().MulScalar(float64(s))) is all axis
 				m = m.RotateTo(up, Axis(a).Vector().MulScalar(float64(s))) //rotation matrix in all axis(4 by 4)
-				Rotations = append(Rotations, m) // 24 rotation matrices
+
+				if a == 1 {
```

Comment:
axis would be better here

Reply:
Thanks. Agreed, renaming that var would aid code reading. I shall rename the var `a` as `axisId` to avoid a potential naming clash on the line above if I call it just `axis` (`Axis(axis)`)

Reply:
Done, thanks.